vendor/engines/saml_authentication/lib/saml_authentication/idp_entity_id_reader.rb

```diff
@@ -0,0 +1,40 @@
+module SamlAuthentication
+
+  # The default IdpEntityIdReader triggers an error on SLO because that request to
+  # us has a SAMLResponse parameter, which does not get parsed by OneLogin::RubySaml::Response
+  # correctly: "Issuer of the Response not found or multiple."
+  # Overrides DeviseSamlAuthenticatable::DefaultIdpEntityIdReader.
```

Comment:
I would put this line at the top of the comments (have the explanation of why we're doing this override below the fact that we are doing it).

Also, do you want to put a note in the comments that we're probably eventually going to open a PR/issue with the gem, and this will hopefully then be able to be taken out?

Reply:
Good calls on all the comment suggestions. I'm probably not going to merge this for a little bit (definitely not until I hear back from DC on their configuration), but I'll address your thoughts.